java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -221,10 +221,10 @@ class EnhancedForStmt extends Stmt, @enhancedforstmt {
   /** Gets the body of this enhanced `for` loop. */
   Stmt getStmt() { result.getParent() = this }
 
-  /** Gets a printable representation of this statement. May include more detail than `toString()`. */
   override string pp() { result = "for (...) " + this.getStmt().pp() }
```

Comment:
Should this be changed to `for (... : ...)` as well?

Reply:
Sure, I think that would be fine.

Reply:
Ok, have changed it in the latest force-pushed commit.